### PR TITLE
Create argument to sort AIPs 

### PR DESCRIPTION
**Pull request by Dhwaniartefact**
This PR creates an argument `--sort` to display the results of fixity check error messages on top and success messages below.

Connected to https://github.com/archivematica/Issues/issues/1681

### CODE REVIEW

File: tests/test_fixity.py [hunk 1]
```diff
@@ -551,3 +470,72 @@ def test_main_handles_exceptions_if_scanall_fails(_get, monkeypatch):
...
+    response = fixity.main(["scanall", "--sort"], stream=stream)
+
+    assert response == 0
+
+    _assert_stream_content_matches(
+        stream,
+        [
+            f"Fixity scan succeeded for AIP: {aip1_uuid}",
+            f"Fixity scan succeeded for AIP: {aip2_uuid}",
+            "Successfully scanned 2 AIPs",
```
Comment (replaceafill) [inline tests/test_fixity.py]: Could you please add a couple of errors to this test case? Something along the lines of:

* Fixity scan of aip1 succeeds
* Fixity scan of aip2 fails
* Fixity scan of aip3 succeeds
* Fixity scan of aip4 fails

And check the output should list them as:

* Fixity scan of aip2 failed
* Fixity scan of aip4 failed
* Fixity scan of aip1 succeeded
* Fixity scan of aip3 succeeded
* Successfully scanned 4 AIPs

**Review by replaceafill — CHANGES_REQUESTED** [review]
This looks good @Dhwaniartefact, but I think we should extend that new test case to assert it's actually separating errors and successes as you describe.

**Review by replaceafill — APPROVED** [review]

**Comment by amayita** [comment]
LGTM Please go ahead and merge!

**Review by amayita — COMMENTED** [review]
LGTM

**Pull request merged**